api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java

```diff
@@ -88,6 +88,36 @@ public void setEncounterDAO(EncounterDAO dao) {
 		    dao.getEncounters(query, null, null, null, includeVoided), null);
 	}
 	
+
+	private void failIfDeniedToEdit(Encounter encounter) throws APIException {
+		throw new APIException("Encounter.error.privilege.required.edit", new Object[] { encounter.getEncounterType()
```

Comment:
i think the changes in this file are not part of the issue. this comes from your other PR. please fix that.